packages/communication-react/CHANGELOG.md

```diff
@@ -6,23 +6,21 @@ This log was last generated on Mon, 13 Jun 2022 18:29:25 GMT and should not be m
 
 ## [1.3.0](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.3.0)
 
+This stable release is mostly minor changes and bug fixes, there are no major feature introductions.
```

Comment:
nit/suggestion: a common, nice addition when nothing major has change is "quality of life improvements"
```suggestion
This stable release contains minor changes, bug fixes and quality of life improvements, there are no major feature introductions.
```